setup.cfg

```diff
@@ -2,3 +2,9 @@
 with-doctest=1
 doctest-options=+ELLIPSIS,+IGNORE_EXCEPTION_DETAIL
 doctest-extension=rst
+with-coverage=1
+ignore-files=tube.py
```

Comment:
Nope. Not sure what's going on here. When I run `nosetests` on this branch I get a strange error about a missing command. But if I remove this line, all the tests run and I see 100% coverage.